### PR TITLE
Remove KFP Python DSL export option as temporary workaround

### PR DESCRIPTION
**Pull request by kiersten-stokes**
This would be a temporary workaround for #1756 and #1760. More discussion needed.

### What changes were proposed in this pull request?
EDITED: It removes the option to export a KFP pipeline in Python DSL format **in Kubeflow specific pipelines** to avoid errors such as those encountered in the above-mentioned issues. 

Kubeflow Pipeline export dialog
Before:
Screen Shot 2021-06-15 at 1 38 10 PM

After:
Screen Shot 2021-06-15 at 1 37 59 PM

Generic Pipeline export dialog remains the same:
image

### How was this pull request tested?
Tests not added due to the temporary nature of this PR.

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by ptitzler** [comment]
Shouldn't do becasue it would impact generic pipelines, for which export should still work as intended.

**Comment by kiersten-stokes** [comment]
@ptitzler makes sense! Thanks, @karlaspuldaro, for the help!

File: packages/pipeline-editor/src/PipelineExportDialog.tsx [hunk 1]
```diff
@@ -74,6 +74,10 @@ export class PipelineExportDialog extends React.Component<IProps, IState> {
     if (!platformSelection) {
       return new Array<Record<string, string>>();
     } else if (platformSelection === KFP_SCHEMA) {
+      // TODO: remove temporary workaround for KFP Python DSL export option
```
Comment (ptitzler) [inline packages/pipeline-editor/src/PipelineExportDialog.tsx:77]: Could we augment this with a link to the issue to provide context information? As is, it's not clear what prompted the TODO and a developer would have to inspect the file history to determine why the temporary change was made.

**Review by ptitzler — APPROVED** [review]
Tested, LGTM

File: packages/pipeline-editor/src/PipelineExportDialog.tsx (hunk 1 above)
Comment (kiersten-stokes) [reply packages/pipeline-editor/src/PipelineExportDialog.tsx:77]: That's a good idea. I'll add that now

**Pull request merged**